client/settings/general-settings-section/disable-upe-confirmation-modal.js

```diff
@@ -0,0 +1,189 @@
+import { Button, ExternalLink } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import ConfirmationModal from 'wcstripe/components/confirmation-modal';
+import InlineNotice from 'wcstripe/components/inline-notice';
+import PaymentMethodsMap from '../../payment-methods-map';
+import UpeToggleContext from '../upe-toggle/context';
+import { useEnabledPaymentMethods } from './data-mock';
+
+const DeactivatingPaymentMethodsList = styled.ul`
+	min-height: 150px;
+
+	> * {
+		&:not( :last-child ) {
+			margin-bottom: $grid-unit-10;
+		}
+	}
+`;
+
+const PaymentMethodListItemContent = styled.div`
+	display: inline-flex;
+	align-items: center;
+	vertical-align: middle;
+	flex-wrap: nowrap;
+
+	> * {
+		margin-right: 4px;
+		line-height: 1em;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+`;
+
+const DisableUpeConfirmationModal = ( { onClose } ) => {
+	const { status, setIsUpeEnabled } = useContext( UpeToggleContext );
+
+	const { createErrorNotice, createSuccessNotice } = useDispatch(
+		'core/notices'
+	);
+
+	const handleConfirmation = () => {
+		const callback = async () => {
+			try {
+				await setIsUpeEnabled( false );
+				createSuccessNotice(
+					__(
+						'What made you disable the new payments experience?',
+						'woocommerce-gateway-stripe'
+					),
+					{
+						actions: [
+							{
+								label: __(
+									'Share feedback (1 min)',
+									'woocommerce-gateway-stripe'
+								),
+								url:
+									'https://woocommerce.survey.fm/woocommerce-stripe-upe-opt-out-survey',
+							},
+						],
+					}
+				);
+				onClose();
+			} catch ( err ) {
+				createErrorNotice(
+					__(
+						'There was an error disabling the new payment methods.',
+						'woocommerce-gateway-stripe'
+					)
+				);
+			}
+		};
+
+		// creating a separate callback so that the UI isn't blocked by the async call.
+		callback();
+	};
+
+	const [ enabledPaymentMethodIds ] = useEnabledPaymentMethods();
+	const upePaymentMethods = enabledPaymentMethodIds.filter(
+		( method ) => method !== 'card'
+	);
+
+	return (
+		<>
+			<ConfirmationModal
+				title={ __(
+					'Disable the new payments experience',
+					'woocommerce-gateway-stripe'
+				) }
+				onRequestClose={ onClose }
+				actions={
+					<>
+						<Button
+							isSecondary
+							disabled={ status === 'pending' }
+							onClick={ onClose }
+						>
+							{ __( 'Cancel', 'woocommerce-gateway-stripe' ) }
+						</Button>
+						<Button
+							isPrimary
+							isDestructive
+							isBusy={ status === 'pending' }
+							disabled={ status === 'pending' }
+							onClick={ handleConfirmation }
+						>
+							{ __( 'Disable', 'woocommerce-gateway-stripe' ) }
+						</Button>
+					</>
+				}
+			>
+				<p>
+					{ __(
+						'Without the new payments experience, your customers will only be able to pay using credit card / debit card. You will not be able to add other sales-boosting payment methods anymore.',
```

Comment:
This text does not match the one in the design.
The one in the designs says: "Without the new payments experience, your customers will no longer be able to pay using the payment methods listed below.". But that clearly doesn't work for when there are no UPE payment methods enabled.